runtime/doc/starting.txt

```diff
@@ -445,31 +447,42 @@ accordingly.  Vim proceeds in this order:
 	commands from the command line have not been executed yet.  You can
 	use "--cmd 'set noloadplugins'" |--cmd|.
 
-5. Set 'shellpipe' and 'shellredir'
+5. Enable filetype plugins and indent scripts.
```

Comment:
It does, the documentation is wrong (in the original version it was like documented here, but the same thing was pointed out -- I guess I forgot to update the docs). Nice catch!